lib/gladiator.rb

```diff
@@ -1,2 +1,28 @@
 class Gladiator
+
+attr_accessor :name, :weapon, :available_weapons
+
+# A gladiator has a name
+  def initialize name = nil, weapon = nil
+    @name = name
+    @weapon = weapon
+  end
+
+  def name
+    return @name
```

Comment:
your `attr_accessor` already creates the `getter` methodds for `name` and `weapon` already